collectors/auth0/utils.js

```diff
@@ -1,7 +1,9 @@
+const moment = require('moment');
+
 function getAPILogs(auth0Client, state, accumulator, maxPagesPerInvocation) {
     let pageCount = 0;
 
-    let params = state.last_log_id ? { from: state.last_log_id, take:100 } : { q: "date=[" + state.since + " TO *]", per_page:100, sort: "date:1" };
+    let params = state.last_log_id ? { from: state.last_log_id, take:100 } : { q: "date=[" + moment(state.since).toISOString() + " TO *]", per_page:100, sort: "date:1" };
```

Comment:
Should the query use `date:[...` instead of `date=[...`, like this?

```suggestion
    let params = state.last_log_id ? { from: state.last_log_id, take:100 } : { q: "date:[" + moment(state.since).toISOString() + " TO *]", per_page:100, sort: "date:1" };
```

From Auth0 docs (https://auth0.com/docs/logs/log-search-query-syntax#example-queries):
Search for all logs from December 10, 2018 forward | date:[2018-12-10 TO *]
-- | --